### PR TITLE
Feature/checkpoint loads all

### PR DESCRIPTION
**Pull request by ditwoo**
## Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md), Pull Request section?
- [x] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [ ] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [ ] Did you check the docs with `make check-docs`?
- [ ] Did you write any new necessary tests?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

You can use 'Login as guest' to see Teamcity build logs.

### CODE REVIEW

File: bin/tests/check_dl_all.sh [hunk 1]
```diff
@@ -14,3 +14,7 @@ bash ./bin/tests/check_dl_cv.sh
 
 ####################################  NLP  ####################################
 bash ./bin/tests/check_dl_nlp.sh
+
+
+#################################  Callbacks  #################################
+bash ./bin/tests/check_dl_callbacks.sh
```
Comment (Scitator) [inline bin/tests/check_dl_all.sh:20]: how do these test differ from https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_core_callbacks.sh ?

Comment (Scitator) [reply bin/tests/check_dl_all.sh:20]: and could we rewrite them into `check_dl_core{something}.sh`, so we could automatically use them https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_all.sh#L8 ?

**Review by Scitator — COMMENTED** [review]
could you please merge with master? somehow the are many duplicate changes :)

File: bin/tests/check_dl_all.sh (hunk 1 above)
Comment (ditwoo) [reply bin/tests/check_dl_all.sh:20]: > how do these test differ from https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_core_callbacks.sh ?

They are the same cause I created a branch from branch used in #784.

> and could we rewrite them into `check_dl_core{something}.sh`, so we could automatically use them https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_all.sh#L8 ?

Sure, I'll update file name.

**Pull request closed without merging**